process/datadog_checks/process/process.py

```diff
@@ -180,7 +180,7 @@ def find_pids(self, name, search_string, exact_match, ignore_ad=True):
                         # As the process list isn't necessarily scanned right after it's created
                         # (since we're using a shared cache), there can be cases where processes
                         # in the list are dead when an instance of the check tries to scan them.
-                        self.log.debug('Process disappeared while scanning')
+                        self.log.exception('Process disappeared while scanning')
```

Comment:
There are some unrelated changes like thos one that slipped through.

Reply:
Will move these QOL changes to a new PR